troupon/static/js/base_scripts.js

```diff
@@ -0,0 +1,162 @@
+
+
+$(document).ready(function() {
+
+
+	//----------------------------------------
+	//	Modals:
+	//----------------------------------------
+
+	function whichTransitionEndEvent(){
+		// utility function for determining TransitionEndEvent browser prefixes:
+		var t,
+			el = document.createElement('fakeelement'),
+	  		transitions = {
+				'transition':'transitionend',
+				'otransition': 'otransitionend',
+				'OTransition':'oTransitionEnd',
+				'MozTransition':'transitionend',
+				'WebkitTransition':'webkitTransitionEnd'
+		  	};
+
+	  	for(t in transitions){
+			if( el.style[t] !== undefined ){
+			  return transitions[t];
+			}
+	  	}
+	}
+
+	var transitionEndEvent = whichTransitionEndEvent(),
+		modalsWrapper = $('.modals-wrapper'),
+		modalsParent = $('.modals-parent'),
+		modalsCloseBtn = $('#modals-close'),
+		openedModal = null;
+
+	const ESC = '27';
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).